layouts/default.vue

```diff
@@ -0,0 +1,18 @@
+<script setup>
+import { ElContainer, ElHeader, ElMain, ElFooter } from "element-plus";
```

Comment:
you could set up an auto import for these components. just update vite section in the nuxt.config

https://element-plus.org/en-US/guide/quickstart.html#on-demand-import